decidim-admin/app/commands/decidim/admin/create_participatory_process_admin.rb

```diff
@@ -35,19 +35,39 @@ def call
 
       attr_reader :form, :participatory_process
 
-      def create_participatory_process_admin
-        ParticipatoryProcessUserRole.create!(
-          role: :admin,
-          user: user,
-          participatory_process: @participatory_process
-        )
+      def create_role
+        if form.roles.include?("collaborator")
+          user.roles << "collaborator" unless user.roles.include?("collaborator")
```

Comment:
See the code [here](https://github.com/AjuntamentdeBarcelona/decidim/blob/master/decidim-admin/app/models/decidim/admin/abilities/collaborator_user.rb#L18).